src/inspector/view/qml/MuseScore/Inspector/general/playback/PlaybackGeneralSettings.qml

```diff
@@ -47,62 +54,122 @@ Item {
 
         spacing: 12
 
+        Repeater {
+            id: repeater
+
+            model: [
+                { componentRole: noteSection, activeRole: root.noteModel ? !root.noteModel.isEmpty : false },
```

Comment:
We can improve this model a bit. Let's rewrite it as follows:

```
model : [
    { typeRole: Inspector.TYPE_NOTE, componentRole: noteSection },
    ...
]
```

Loader will look like this:

```
Loader {
    id: expandableLoader

    property var itemModel: root.proxyModel ?  proxyModel.modelByType(modelData.typeRole) : null

    sourceComponent: modelData["componentRole"]
    active: Boolean(itemModel) && !itemModel.isEmpty
    
    onLoaded: {
        expandableLoader.item.model = expandableLoader.itemModel
    }
}
```

This will help reduce code duplication and remove the models from the root level